CROWDIN.md

```diff
@@ -0,0 +1,62 @@
+# Crowdin Translator How-To
+
+Stellar Developer Docs use [Crowdin](https://crowdin.com/) for localization. This document provides steps and resources for managing Stellar Docusaurus Crowdin translations as a Crowdin user with a Translator role.
+
+---
+
+## Overview
+
+As new English source files are added to the Stellar Docusaurus Documentation GitHub repository, they are automatically uploaded to Crowdin. From there, an automated AI translation is run on the files to translate them into other target languages (currently only Spanish at the time of writing).
+
+In cases where the AI translation is inaccurate, a user with the Translator role will need to update and edit the translations manually.
+
+---
+
+## Translator User
+
+To get started, you need to set up a Crowdin account and be added to the Stellar Documents Crowdin project. This involves creating a Crowdin user account and requesting access to the project from a project owner. 
```

Comment:
theres a project setting that we could use to configure this, but I _think_ right now the project is "open" and anybody could join. We could enable "member moderation" (or it's called something similar), though, and we'd have to approve anybody who wants to join.